queries/schedules.py

```diff
@@ -25,6 +25,7 @@
 """
 
 # TODO: Support TV schedule queries for other stations than RÚV
+#       (the endpoints currently seem to be broken on apis.is, as of June 1st 2021)
```

Comment:
Might want to take a look at how the apis.is guys were fetching this info? Maybe it makes more sense to talk directly to the source instead of proxying through apis.is?